src/app.service.ts

```diff
@@ -1,8 +1,11 @@
 import { Injectable } from '@nestjs/common';
 
 @Injectable()
-export class AppService {
+export default class AppService {
+
+  private readonly helloWorld = 'Hello World!';
+
   getHello(): string {
-    return 'Hello World!';
+    return this.helloWorld;
```

Comment:
Can we revert this change to stay consistent with all tutorials & guides & courses on the web?

Reply:
So you would like to turn off this eslint rule?
https://eslint.org/docs/rules/class-methods-use-this